instrumentation-security/spring-webmvc-5.3.0/src/main/java/org/springframework/web/servlet/handler530/SpringHelper.java

```diff
@@ -0,0 +1,35 @@
+package org.springframework.web.servlet.handler530;
+
+import com.newrelic.api.agent.security.NewRelicSecurity;
+import com.newrelic.api.agent.security.schema.ApplicationURLMapping;
+import com.newrelic.api.agent.security.instrumentation.helpers.*;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.servlet.mvc.condition.PathPatternsRequestCondition;
+import org.springframework.web.servlet.mvc.condition.PatternsRequestCondition;
+import org.springframework.web.servlet.mvc.method.RequestMappingInfo;
+import org.springframework.web.util.pattern.PathPattern;
+
+import java.util.Iterator;
+
+public class SpringHelper {
+    public static <T> void gatherURLMappings(T mapping){
+        try {
+            RequestMappingInfo mappingInfo = (RequestMappingInfo) mapping;
+            for (RequestMethod method : mappingInfo.getMethodsCondition().getMethods()) {
+                PatternsRequestCondition patternsCondition = mappingInfo.getPatternsCondition();
+                PathPatternsRequestCondition pathPatternsCondition = mappingInfo.getPathPatternsCondition();
```

Comment:
Extract the two statements out of for loop

Reply:
Fixed this in commit [a89078f](https://github.com/newrelic/csec-java-agent/pull/53/commits/a89078fac42fdd90722642f69c2063d23227024c)